src/plugins/console/public/services/autocomplete.ts

```diff
@@ -20,6 +21,8 @@ import {
 } from '../lib/autocomplete_entities';
 import { DevToolsSettings, Settings } from './settings';
 
+const AUTOCOMPLETE_TIMEOUT = 30000;
```

Comment:
I vote we have a limit on the number of records returned instead of a timeout. In addition, if we still continue with a timeout, it must be on the server side so that we have the opportunity to abort requests that are too long